src/uWS.h

```diff
@@ -1,11 +1,30 @@
 #ifndef UWS_H
 #define UWS_H
 
+#if defined(_WIN32) && defined(__cplusplus_cli)
+#define HAVE_WIN32_CPP_CLI 1
```

Comment:
No reason in particular, just to imply that this was Windows related. Shall i remove it?